webapp/src/jaas/reducers/store.ts

```diff
@@ -0,0 +1,13 @@
+import {applyMiddleware, createStore, Store} from 'redux';
+import {loadInitialState, JaaSState} from '../util';
```

Comment:
Was there any comment on the Mattermost PR regarding why the `jass` folder contains separate directories for everything like `components`, `reducers`, `actions` etc.

Reply:
No there was no such comment

Reply:
Should we merge the folders inside `jaas` with the folders outside? (This will be a TODO that we'll do after creating a PR in Mattermost)